pandas/core/internals/blocks.py

```diff
@@ -177,6 +177,9 @@ def get_values(self, dtype=None):
             return self.values.astype(object)
         return self.values
 
+    def get_block_values(self, dtype=None):
```

Comment:
we really need this?

Reply:
is this still needed?

Reply:
Well, the JSON C code *does* handle blocks (if I remove the `get_block_values` part in objToJSON.C that I introduced in this PR, a couple tests fail). 
I could name this here the same as for Series/INdex (i.e. `_internal_get_values`), but I prefer a distinct name to make it clear that the json code is handling blocks and not series/index (and that also makes it clear that all other places where `_internal_get_values` is used is *not* handling blocks). That will also make it easier to isolate and try to remove the block handling in the json C code.

Reply:
Opened https://github.com/pandas-dev/pandas/issues/27164 for follow-up on this JSON issue